arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts

```diff
@@ -660,6 +666,7 @@
 			vdd18s5run@47 {
 				//P0_VDD_18_S5_RUN
 				compatible = "pmbus";
+				reg = <0x47>;
```

Comment:
This change is unrelated to the commit message. Please update either the commit message or split the patch.